distributed/tensor_parallelism/two_d_parallel_example.py

```diff
@@ -2,24 +2,23 @@
 
 import torch
 import torch.distributed as dist
-import torch.multiprocessing as mp
 
 from torch.distributed._tensor import DeviceMesh
 from torch.distributed.fsdp import FullyShardedDataParallel as FSDP
 from torch.distributed.tensor.parallel import (
     PairwiseParallel,
     parallelize_module,
+    ColwiseParallel,
+    RowwiseParallel,
 )
-from torch.distributed.tensor.parallel.fsdp import enable_2d_with_fsdp
 
-from utils import cleanup, setup, ToyModel
-try:
-    from torch.distributed.tensor.parallel import (
-        SequenceParallel
-    )
-    SP_AVAILABLE = True
-except BaseException as e:
-    pass
+
+from torch.distributed._shard.sharded_tensor import ShardedTensor
+from torch.distributed._tensor import DTensor, Replicate, sharding_prop
+from torch.distributed._tensor.device_mesh import init_device_mesh
+import os
+
+from utils import cleanup, torchrun_setup, MLP_swiglu
```

Comment:
Let's define MLP_swiglu in each example file, it's fine to have some duplications as these are examples, we want user to see e2e workflow in a single file